components/BrandedButton/BrandedButton.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <component
-    :is="as"
+    :is="href ? 'a': 'button'"
```

Comment:
I think we should use a `NuxtLinkLocale` here instead of a `a` to avoid a full page load

Reply:
I don't know how to do it… https://github.com/nuxt-modules/i18n/issues/3257

Reply:
I didn't find the correct import for `NuxtLinkLocale`

Reply:
The correct import is `import { NuxtLinkLocale } from '#components'` but it is indeed quite hard to find and not autocompleted